nep-11.mediawiki

```diff
@@ -0,0 +1,176 @@
+<pre>
+  NEP: 11
+  Title: Non-Fungible Token Standard
+  Author: Joe Stewart <hal0x2328@splyse.tech>, Shane Mann <fetter@splyse.tech>
+  Type: Standard
+  Status: Accepted
+  Created: 2018-04-18
+</pre>
+
+==Abstract==
+
+This NEP defines a standard non-fungible token system for the NEO Smart Economy.
+
+==Motivation==
+
+NFTs are required to track, exchange and enforce ownership of digital assets. A non-fungible token (NFT) can be thought of like a property deed - each one is unique and carries some non-mutable information (e.g. the physical address of the property). Other information, such as the owner of the property, can be changed. Also, we provide a built-in optional divisibility within each non-fungible asset. This allows for high value objects to be tokenized more effectively.
+
+==Specification==
+
+===Methods===
+
+====name====
+
+<pre>
+public static string name()
+</pre>
+
+Returns the name of the token. e.g. <code>"MyNFT"</code>. 
+
+This method MUST always return the same value every time it is invoked. 
+
+====symbol====
+
+<pre>
+public static string symbol()
+</pre>
+
+Returns a short string symbol of the token managed in this contract. e.g. <code>"MNFT"</code>. This symbol SHOULD be short (3-8 characters is recommended), with no whitespace characters or new-lines and SHOULD be limited to the uppercase latin alphabet (i.e. the 26 letters used in English). 
+
+This method MUST always return the same value every time it is invoked. 
+
+====totalSupply====
+
+<pre>
+public static BigInteger totalSupply()
+</pre>
+
+Returns the total token supply deployed in the system.
+
+====decimals====
+
+<pre>
+public static byte decimals()
```

Comment:
In case that one car is divisible and another one is not, how would we track each corresponding part?

Reply:
A car is divisible, not that it really cuts the car. Dividable cars mean they are shared by many people. So why one car is divisible and another one is not?

Reply:
Let's change example.

That NFT can represent the right that the owner has with someone.
This right can be divisible.

But, the same NFT can have both cases depending on the demand and offer of the market.

Reply:
I mean, shares of a NFT.